tests/foreman/api/test_host.py

```diff
@@ -135,11 +128,35 @@ def test_positive_get_per_page(self):
             data={'per_page': per_page},
             verify=False,
         )
-        self.assertEqual(response.status_code, http.client.OK)
-        self.assertEqual(response.json()['per_page'], per_page)
+        assert response.status_code == http.client.OK
+        assert response.json()['per_page'] == per_page
+
+    @tier2
+    def test_positive_search_by_org_id(self):
+        """Search for host by specifying host's organization id
+
+        :id: 56353f7c-b77e-4b6c-9ec3-51b58f9a18d8
+
+        :customerscenario: true
+
+        :expectedresults: Search functionality works as expected and correct
```

Comment:
Good if we can be specific instead of generic here.

Reply:
Test have 6 lines. It only search for a host, and comment in the test explains why the `organization id` is shown.

Reply:
Jitendra is right, this is too generic. This isn't a test of the whole search functionality so you can't expect that everything search-related works. Suggestion: 'Correct host found by search by its id'

Reply:
DONE.